src/test/scala/com/exasol/avro/AvroPrimitiveReaderTest.scala

```diff
@@ -74,15 +74,15 @@ class AvroPrimitiveReaderTest extends AnyFunSuite {
     val thrown = intercept[IllegalArgumentException] {
       unionTest(schema)
     }
-    assert(thrown.getMessage === "Avro Union type should contain a primitive and null!")
+    assert(thrown.getMessage() === "Avro Union type should contain a primitive and null!")
   }
 
   test("parse avro union without a null type") {
     val schema = getSchema(s"""["string", "int"]""")
     val thrown = intercept[IllegalArgumentException] {
       unionTest(schema)
     }
-    assert(thrown.getMessage === "Avro Union type should contain a primitive and null!")
+    assert(thrown.getMessage() === "Avro Union type should contain a primitive and null!")
```

Comment:
Will it throw nice error messages? (it's just assert, not hamcrest or so)

Reply:
ah good. But by actual question was if the way of asserting gives a nice output in scala

Reply:
Yes, for this it should be something like this:
```
[info] - mkString returns separated key-value property pairs *** FAILED ***                                                                                                                     
[info]   "a[;1 -> k1;v1 -> k2;]v2" did not equal "a[ -> 1;k1 -> v1;k2 -> ]v2" (AbstractPropertiesTest.scala:109)                                                                                
[info]   Analysis:                                                                                                                                                                              
[info]   "a[;1 -> k1;v1 -> k2;]v2" -> "a[ -> 1;k1 -> v1;k2 -> ]v2"
```